.github/pull_request_template.md

```diff
@@ -0,0 +1,10 @@
+Please refer to [CONTRIBUTING.md](CONTRIBUTING.md) for detailed instructions.
+
+## Summary
+<!-- Provide a short summary of your changes -->
+
+
+## PR Checklist
+- [ ] My PR is less than 500 lines of code
+- [ ] I have added sufficient comment as docstrings in my code
+- [ ] I have made corresponding changes to the documentation
```

Comment:
Could we add a check box for writing unit tests?

Reply:
Good idea. However, that will be enforced by automated checks once we have a coverage tool set-up, so it'll be redundant to add it here. But, I guess until we have that tool, we can have it as a checklist.

Reply:
That would be great! But right now we don't have full coverage with the unit tests so will have to work for that